CMakeLists.txt

```diff
@@ -18,28 +19,19 @@ ExternalProject_Add_Step(pybind11_src CopyToDevel
   DEPENDEES install
 )
 
-if(IS_DIRECTORY ${CMAKE_INSTALL_PREFIX})
-  message(WARNING "Compiling for install workspace (${CMAKE_INSTALL_PREFIX})")
-  catkin_package(
-    INCLUDE_DIRS include/${PROJECT_NAME}
-    LIBRARIES ${PYTHON_LIBRARIES}
-    CFG_EXTRAS pybind11_catkin.cmake
-  )
-else()
-  message(WARNING "Compiling for devel workspace")
-  file(MAKE_DIRECTORY ${CATKIN_DEVEL_PREFIX}/include/${PROJECT_NAME})
-  catkin_package(
-    INCLUDE_DIRS ${CATKIN_DEVEL_PREFIX}/include ${EIGEN3_INCLUDE_DIR}
-    LIBRARIES ${PYTHON_LIBRARIES}
-    CFG_EXTRAS pybind11_catkin.cmake
-  )
-endif()
+file(MAKE_DIRECTORY ${CATKIN_DEVEL_PREFIX}/include/${PROJECT_NAME})
```

Comment:
I think this may fail on the ROS buildfarm (no root access during the first build) which is why I added the weird if-logic in the first place.

Reply:
I don't see why. A normal user should have write access to CATKIN_DEVEL_PREFIX. Actually, pybind11 is installed into a subfolder of it.